src/api/db/models/Image.ts

```diff
@@ -181,14 +181,14 @@ export class ImageModel {
   static async createImage(
     input: gql.CreateImageInput,
     context: Pick<Context, 'user'>,
-  ): Promise<HydratedDocument<ImageAttemptSchema> & { errs?: ImageErrorSchema[] }> {
+  ): Promise<gql.ImageAttempt> {
```

Comment:
It may be preferable to return GraphQL codegen-generated types in more model methods because (a) we don't need all the properties and methods that a Mongoose `HydratedDocument` contains and we also won't have to extend the `HydratedDocument`s so much with any additional properties that we tack on after fetching the record from the DB.